tarsier-snapshots/tarsier_snapshots/snapshots.py

```diff
@@ -63,8 +64,12 @@ async def snapshot_example(
         print(f"{prefix} Snapshotting {example.id}")
         await page.goto(example.get_static_url())
         await page.wait_for_timeout(3000)
+        start_time = time.time()
         image, _ = await tarsier.page_to_image(page, tag_text_elements=True)
+        screenshot_duration = time.time() - start_time
+        start_time = time.time()
         page_text, _ = await tarsier.page_to_text(page, tag_text_elements=True)
+        ocr_duration = time.time() - start_time
```

Comment:
When doing benchmarking it is usually better to use `time.perf_counter()` instead of `time.time()`. eg;
```python
import time

# Start the timer
start_time = time.perf_counter()

# Code block to measure
# Example: some computation or processing
for i in range(1000000):
    pass

# End the timer
end_time = time.perf_counter()

# Calculate the elapsed time
elapsed_time = end_time - start_time

print(f"Elapsed time: {elapsed_time} seconds")
```